src/Microsoft.Sbom.Targets/README.md

```diff
@@ -0,0 +1,55 @@
+
+Finally, the `Microsoft.Sbom.Targets.targets` file creates a target that will execute the custom MSBuild task. This file will be automatically imported when consuming the NuGet package.
+
+## SBOM Generation Properties
+The custom MSBuild task accepts most of the arguments available for the [SBOM CLI Tool](../../docs/sbom-tool-arguments.md). After the .targets file is imported into a .NET project, the following properties can be set:
+
+| Property | Default Value | Required |
+|-----------------------------------------------------|-------------|---------|
+| `<GenerateSBOM>`                                    | `false`     | No. To enable SBOM generation, set this to true. |
+| `<SbomGenerationBuildDropPath>`                     | `$(OutDir)` | Yes |
+| `<SbomGenerationBuildComponentPath>`                | `$(MSBuildProjectDirectory)` | No | 
+| `<SbomGenerationPackageSupplier>`                   | `$(Authors)`. If `$(Authors)` is null, it will set `$(AssemblyName)`     | Yes | 
+| `<SbomGenerationPackageName>`                       | `$(PackageId)`. If `$(PackageId)` is null, it will set `$(AssemblyName)` | Yes | 
+| `<SbomGenerationPackageVersion>`                    | `$(Version)`. If `$(Version)` is null, it will set "1.0.0"               | Yes | 
+| `<SbomGenerationNamespaceBaseUri>`                  | `http://spdx.org/spdxdocs/$(SbomGenerationPackageName)`                  | Yes | 
+| `<SbomGenerationNamespaceUriUniquePart>`            | N/A | No | 
+| `<SbomGenerationExternalDocumentReferenceListFile>` | N/A | No | 
+| `<SbomGenerationFetchLicenseInformation>`           | `false` | No | 
+| `<SbomGenerationEnablePackageMetadataParsing>`      | `false` | No | 
+| `<SbomGenerationVerbosity>`                         | `Information` | No | 
+| `<SbomGenerationManifestInfo>`                      | `SPDX:2.2` | No | 
+| `<SbomGenerationDeleteManifestDirIfPresent>`        | `true` | No | 
+| `<SbomGenerationManifestDirPath>`                   | N/A | No | 
+
+## Local SBOM Generation Workflow
+After building the Microsoft.Sbom.Targets project, it will generate a NuGet package containing the MSBuild task's binaries and associated .targets file in the `bin\$(Configuration)` folder. The following steps describe how to consume this NuGet package and generate an SBOM:
+
+1) Create a sample .NET project.
+2) Open the project's NuGet package manager.
+3) Add the path to the Microsoft.Sbom.Targets NuGet package as a package source. You can name it "Local". 
+4) Look for the Microsoft.Sbom.Targets package within the package manager and install it. 
+5) Add the following to your sample project's .csproj file:
+```
+<PropertyGroup>
+  <GenerateSBOM>true</GenerateSBOM>
+</PropertyGroup>
+```
+6) Build the sample project. An SBOM should be generated under the `bin\$(Configuration)\$(TargetFramework)` folder. 
```

Comment:
Delete the second sentence "An SBOM should be generated under the ...."

Reply:
Done